### PR TITLE
[FIX] Remove mass mail upgrade button

### PR DESCRIPTION
**Pull request by mstuttgart**
* [FIX] Remove upgrade button in Mass Mail Settings
* Remove commented code

### CODE REVIEW

File: res_config_settings_enterprise_remove/models/res_config_settings.py [hunk 1]
```diff
@@ -26,40 +26,25 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
...
-                "and @widget='upgrade_boolean']]",
-                "//div[field[@name='module_account_sepa' and \
-                                    @widget='upgrade_boolean']]",
-                "//div[field[@name='module_l10n_us_check_printing' and \
-                                    @widget='upgrade_boolean']]",
-                "//div[field[@name='module_account_reports_followup' and \
-                                    @widget='upgrade_boolean']]",
-                "//div[field[@name='module_account_batch_deposit' and "
-                "@widget='upgrade_boolean']]",
+                "//div[field[@name='module_account_reports' and @widget='upgrade_boolean']]",  # noqa: E501
```
Comment (pedrobaeza) [inline res_config_settings_enterprise_remove/models/res_config_settings.py]: Don't change this and label as `noqa`

**Comment by mstuttgart** [comment]
@pedrobaeza fixed.

File: res_config_settings_enterprise_remove/models/res_config_settings.py [hunk 2]
```diff
@@ -85,7 +85,13 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
             queries += [
                 "//label[@for='module_inter_company_rules']",
                 "//div[@name='inter_company']",
-                # "//field[@name='module_project_forecast']/ancestor::div[2]",
```
Comment (pedrobaeza) [inline res_config_settings_enterprise_remove/models/res_config_settings.py:88]: Why do you comment this line?

Comment (mstuttgart) [reply res_config_settings_enterprise_remove/models/res_config_settings.py:88]: I don't comment this line. It already commented.

https://github.com/OCA/server-tools/blob/10.0/res_config_settings_enterprise_remove/models/res_config_settings.py#L88

Comment (pedrobaeza) [reply res_config_settings_enterprise_remove/models/res_config_settings.py:88]: Ah, OK, you're right.

**Review by pedrobaeza — APPROVED** [review]
Code review

**Review by aldo774 — APPROVED** [review]
LGTM :+1:

**Pull request merged**